src/graph.h

```diff
@@ -247,6 +254,10 @@ struct ImplicitDepLoader {
   /// @return false on error (without filling \a err if info is just missing).
   bool LoadDepsFromLog(Edge* edge, string* err);
 
+  /// Insert the loaded implicit dependencies (\a nodes) of \a edge
+  /// into the graph.
+  void ApplyLoadedDeps(Edge* edge, Node** nodes, int node_count);
```

Comment:
node_count should be size_t, that's the representation for "number of objects in memory"

Reply:
DepsLog::Deps node_count is an int though, around src/deps_log.h:84.

Reply:
Alright. That's horrible, but you're right that it should be an int.